docs/current/cli/698277-index.md

```diff
@@ -0,0 +1,63 @@
+---
+slug: /cli
+---
+
+# Dagger CLI
+
+<div class="status-badge">Technical Preview</div>
+
+## What is the Dagger CLI?
+
+The Dagger CLI contains everything you need to develop and run CI/CD pipelines from the command-line.
+
+Currently the CLI consists of:
+
+* A binary file
+* This documentation
+
+## Who is it for?
+
+The Dagger CLI may be a good fit if you are...
+
+*
+
+The Dagger Python SDK may *not* be a good fit if you are...
+
+* A developer who doesn't know shell scripting, and is not interested in learning it.
+* Someone who loves writing YAML all day, thank you very much.
+* A container skeptic: the less containers are involved, the happier you are.
+
+## How does it work?
+
+```mermaid
+graph LR;
+
+subgraph program["Your program"]
+  lib["Dagger CLI"]
+end
+
+engine["Dagger Engine"]
+
+oci["OCI container runtime"]
+subgraph A["your build pipeline"]
+  A1[" "] -.-> A2[" "] -.-> A3[" "]
+end
+subgraph B["your test pipeline"]
+  B1[" "] -.-> B2[" "] -.-> B3[" "] -.-> B4[" "]
+end
+subgraph C["your deployment pipeline"]
+  C1[" "] -.-> C2[" "] -.-> C3[" "] -.-> C4[" "]
+end
+
+lib -..-> engine -..-> oci -..-> A1 & B1 & C1
+```
+
+1. Using the Dagger CLI, your program (typically a shell script) opens a new session to a Dagger Engine: either by connecting to an existing engine, or by provisioning one on-the-fly.
```

Comment:
I've updated this sequence slightly compared to the SDK introductions. Could you please check it once?